### ISSUE TITLE
Sort order for list should have ongoing first, closed or completed at the end.

### ISSUE THREAD

**Issue by gclancy** [esridc/openfda#54]
![image](https://cloud.githubusercontent.com/assets/13022624/8386686/56205692-1c20-11e5-99db-c633d25ed607.png)

**Comment by benheb** [comment]
@ajturner have you come across ability for SORT in FDA API? Or should I just sort whatever results I get back first by status?

**Comment by gclancy** [comment]
SORT order still not working - some completed recalls are showing above ONGOING ones

![image](https://cloud.githubusercontent.com/assets/13022624/8420371/ab394d7a-1e8f-11e5-97c8-60861587befd.png)

**Comment by benheb** [comment]
Correct, this is not done. What's tricky about this, is that I just request the first 100 results from the API based on state, and it returns those with any status. So I could sort those results, but as soon as you scroll and paginate, the order will be mucked again. I'm thinking we might have to leave as unsorted, and if the user wants to see only "ongoing" projects, they can use the filter option above. Does that sound okay? @gclancy @ajturner @michaelgaigg

**Comment by gclancy** [comment]
Determined that we should leave in backlog.  If this had been a real TO we would work with the end users to get additional feedback on SORT order and determine a more complete technical strategy to support.
